### DISCUSSION TITLE
Inexistent ebsdId in grain boundary dataset

### DISCUSSION THREAD

**Discussion by Dxc0000** [mtex-toolbox/mtex#2078] [Ask Anything]
`ebsd = EBSD.load(fname,CS,'interface','crc',...
  'convertEuler2SpatialReferenceFrame');`
`ebsd = ebsd('Ni-superalloy');
[grains,ebsd.grainId] = calcGrains(ebsd);`
`gB = grains.boundary('Ni-superalloy','Ni-superalloy');`
`gBebsdId = gB.esbdId`

It is clear that I want to acquire the ebsdIds across each grain boundary(segment), and calculate their orientations and misorientation.
Here the problem is, the max in exported gBebsdId is larger than the total ebsd number, which is extremely odd. I want to ask why?

**Comment by kilir** [comment]
Hi,
which mtex version are you using?
Cheers,
Rüdiger

**Reply by Dxc0000** [reply]
version5.9.0
Now I understand the situation is that when I calculate grain and grain boundaries, though "Ni-superalloy, Ni-superalloy" is defined, number of ebsd decrease with my selection yet there intrinsic number didn't change, am I right about it?
Since you have mention the version issue, will this problem be  solved with another version?
